src/main/kotlin/com/cosmotech/api/config/CsmPlatformProperties.kt

```diff
@@ -83,10 +83,26 @@ data class CsmPlatformProperties(
     /** Queue configuration */
     val eventbus: CsmEventBus
 ) {
-  data class CsmStorage(val host: String, val reader: CsmStorageUser, val admin: CsmStorageUser) {
+  data class CsmStorage(
+      val host: String,
+      val port: String = "5432",
+      val reader: CsmStorageUser,
+      val writer: CsmStorageUser,
+      val admin: CsmStorageUser
+  ) {
     data class CsmStorageUser(val username: String, val password: String)
   }
-  data class CsmEventBus(val host: String, val username: String, val password: String)
+  data class CsmEventBus(
+      val host: String,
+      val port: String = "5672",
+      val defaultExchange: String = "csm-exchange",
+      val defaultQueue: String = "csm",
+      val defaultRoutingKey: String = "csm",
```

Comment:
This is not something we have configured in the RabbitMQ deployment. I'm not even sure we can or want to do that. We need to dig that config aspect.

Reply:
This is part of current Cosmotech RabbitMqConfig, will see next week to:
-  remove defaultQueue creation 
- change defaultExchange to `amq.default `(which is here as default when rabbitMQ is deployed)
- remove defaultRoutingKey

Reply:
Note: I think `amq.topic` might be more appropriate if the default one does not work.